data/diann_annotation.csv

```diff
@@ -0,0 +1,16 @@
+Run,Tx,Well,Unique Peptides,Plate,Condition,BioReplicate
+230719_THP-1_Chrom_end2end_Plate1_DMSO_A02_DIA,DMSO,A02,45341,Plate1,DMSO,2
+230719_THP-1_Chrom_end2end_Plate1_DMSO_A05_DIA,DMSO,A05,44666,Plate1,DMSO,5
```

Comment:
So this DIANN dataset is actually owned by one of our collaborators, so we can't add it as data here.  Could you use one of the datasets inside the MSstatsConvert package for your vignette?